src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -13,4 +13,16 @@ To see the custom values you used for this release:
 
   $ helm get values {{ .Release.Name }} -n {{ .Release.Namespace }}
 
+{{ if .Values.ingress.create -}}
+Service URL: {{ ternary "https" "http" .Values.ingress.https -}}://{{ .Values.ingress.host }}
+{{- else if contains "ClusterIP"  .Values.bitbucket.service.type -}}
+Get the product URL by running these commands in the same shell:
```

Comment:
Similar to the previous comment:

`Get the Bitbucket service URL by running these commands in the same shell:`